[ADD] crm: unattended leads report doc (#7965)
odoo/documentation: content/applications/sales/crm/track_leads/unattended_leads_report.rst

=======================
Unattended leads report
=======================

*Unattended leads* are leads that have scheduled activities that are either due or past due.

> Whenever an activity is scheduled, *Odoo* tracks the due date and sends email reminders to the users

Reviewer: ```suggestion
Whenever an activity is scheduled, Odoo tracks the due date, and sends email reminders to the users
```

the activity is assigned to.

An *unattended leads report* compiles all active leads in the pipeline with due or past due
activities, allowing a sales manager to identify which opportunities require immediate attention.

By pulling a daily unattended leads report, sales managers can remind their teams to address
outstanding activities before they become past due, helping avoid neglected leads and reinforcing
proactive behaviors in their salespeople.

.. example::

   A sales manager starts their day by pulling an unattended leads report, and upon switching to
   list view, they see the following:

   .. image:: unattended_leads_report/unattended-leads-example.png
      :align: center
      :alt: List view of a sample Unattended Leads report with the activities emphasized.

>   Their team member, Mitchell, has two leads in the Proposition stage with activities that are due.
>   The yellow :guilabel:`📞 (phone)` icon indicates that the `5 VP Chairs` lead has a phone call
>   activity scheduled for today. The red :guilabel:`✉️ (envelope)` icon indicates that the `Modern
>   Open Space` lead has an email activity scheduled that is past due.

Reviewer: ```suggestion
  Their team member, Mitchell, has two leads in the *Proposition* stage with activities that are
  due.
  
  The yellow :guilabel:`📞 (phone)` icon indicates that the `5 VP Chairs` lead has a phone call
  activity scheduled for today. The red :guilabel:`✉️ (envelope)` icon indicates that the `Modern
  Open Space` lead has an email activity scheduled that is past due.
```

  Clicking on the `Modern Open Space` lead, the sales manager opens the record of the lead and
  reviews the chatter. They see that the email was scheduled to be sent two days ago, but Mitchell
  never marked this activity as done.

  .. image:: unattended_leads_report/overdue-activities-email.png
     :align: center
     :alt: Example of overdue activities notification in the chatter of a lead.

.. important::
   In order to pull an unattended leads report, activity track **must** be utilized in the *CRM*
   pipeline.

   For more information, refer to :ref:`Activity Tracking
   <track_leads/unattended_leads_report/activity tracking>`

Create an unattended leads report
=================================

To create an unattended leads report, first navigate to :menuselection:`CRM app --> Reporting -->

> Pipeline` to open the :guilabel:`Pipeline Analysis` report. Click into the search bar at the top of
> the page. The :guilabel:`Created on` filter can remain active, though if any other filters are
> present, they should be removed.

Reviewer 2: - It's not *quite* a report yet (it's a *dashboard* where reports are generated)
- let's label the search bar with a gui and the placeholder text it's been given since it's now part of an active instruction.
- add transitional language
- wording adjustment

```suggestion
Pipeline` to open the :guilabel:`Pipeline Analysis` dashboard. Then, click into the
:guilabel:`Search...` bar at the top of the page, and remove all of the default filters. Note, the
:guilabel:`Created on` filter may remain active, as this variable can be useful to include in the
report.
```

> Click the down arrow to the right of the search bar to open the filters drop-down menu. Click
> :guilabel:`Add Custom Filter`. This opens a :guilabel:`Add Custom Filter` pop-up window.

Reviewer 2: This section is way too curt and does not _guide_ users or adequately describe what they're clicking through. Specifically, we're missing: 
- emoji 🔻
- form coverage / depth of content, 
- gui(s) (🔻button, search bar vs. **Search...** bar)
- grammar (e.g. a vs. an)
- transitional language (first, next, then, lastly, etc.)

```suggestion
Next, add custom filters by clicking the :guilabel:🔻 (triangle pointed down) icon to the right of
the :guilabel:`Search...` bar to open the drop-down mega menu that contains :guilabel:`Filters`,
:guilabel:`Group By`, and :guilabel:`Favourites` columns. Under the :guilabel:`Filters` column, click
:guilabel:`Add Custom Filter`, which opens an :guilabel:`Add Custom Filter` pop-up window.
```

.. important::
   At the top of the :guilabel:`Add Custom Filter` form, there is an option to match :guilabel:`any`
   or :guilabel:`all` of the rules. In order to properly run the report, only records that match
   **all** of the following filters should be included. Before adding the filters, make sure
   :guilabel:`all` is selected in this field.

   .. image:: unattended_leads_report/all-custom-filter.png
     :align: center
     :alt: Example of overdue activities notification in the chatter of a lead.

Add filter for past due activities
----------------------------------

Click the first field for the new rule, and type `Activities` in the search bar, or scroll to search
through the list. Then, next to :guilabel:`Activities`, click the arrow to open a new drop-down menu
with secondary conditions. Select :guilabel:`Due Date`.

  .. image:: unattended_leads_report/activities-due.png
     :align: center
     :alt: Custom filter pop-up with emphasis on the options for activities and due date.

In the rule's second field, select :guilabel:`<=` from the drop-down menu. Selecting this filter

> includes all activities with a due date up to, and including the date selected in the next field.

Reviewer: ```suggestion
includes all activities with a due date up to, and including, the date selected in the next field.
```

Either leave the rule's third field as today's date, or adjust it as needed.

Exclude unassigned leads
------------------------

Click :guilabel:`New Rule`. Click the first field for the new rule, and type `Salesperson` in the
search bar, or scroll to search through the list.

In the rule's second field, select :guilabel:`is set` from the drop-down menu. Selecting this filter

> excludes any leads that have note been assigned to a specific salesperson.

Reviewer: ```suggestion
excludes any leads that have not been assigned to a specific salesperson.
```

Add a Sales team
----------------

Click :guilabel:`New Rule`. Click the first field for the new rule, and type `Sales Team` in the
search bar, or scroll to search through the list.

In the rule's second field, select :guilabel:`is in` from the drop-down menu.

In the third field, enter the name of the desired sales team. Multiple teams can be selected in this
field.

.. note::
   This filter is optional. To view results for the entire company, do **not** add this filter.

.. figure:: unattended_leads_report/configured-custom-rules.png
   :align: center
   :alt: An example of the Custom Filter pop-up window with all of the rules configured.

>    An example of the Add Custom Filter pop-up window with all of the rules configured.

Reviewer: ```suggestion
   An example of the :guilabel:`Add Custom Filter` pop-up window with all of the rules configured.
```

View results
------------

After the filters are configured, click :guilabel:`Add`. The resulting report displays all leads
assigned to a salesperson where an activity is due on the current date, or is past due. The default
display is a bar graph, where the leads are grouped by *stage*.

To group the results by salesperson, click the down arrow to the right of the search bar. Under the
:guilabel:`Group By` heading, select :guilabel:`Salesperson`.

.. note::

>    The option to group by :guilabel:`Sales Team` is available under the :guilabel:`Group By` heading

Reviewer: ```suggestion
   The option to group by :guilabel:`Sales Team` is available under the :guilabel:`Group By` heading,
```

   as well.

To change to a *list* view, click the :guilabel:`≣ (bars)` icon in the top-right corner of the
screen.

.. tip::
   Clicking the :guilabel:`(sliders) icon` opens a drop-down menu of additional columns that can be
   added to the report.

   Some options that are beneficial for this report include:

   - :guilabel:`Activities`: the summary of the latest activity for this lead.
   - :guilabel:`Expected Closing`: estimated date on which the lead will be won.
   - :guilabel:`Probability`: estimated success rate based on the stage.

  .. image:: unattended_leads_report/additional-options.png
     :align: center
     :alt: Custom filter pop-up with emphasis on the options for activities and due date.

.. _track_leads/unattended_leads_report/activity tracking:

Activity tracking
=================

:doc:`Activities </applications/productivity/discuss/overview/plan_activities>` are follow-up tasks
tied to a record in the database. Activities can be scheduled on any page of the database that
contains a chatter thread, Kanban view, list view, or activities view of an application.

> A best practice for pipeline management is scheduling activities or marking them as *done* anytime a

Reviewer: ```suggestion
A best practice for pipeline management is scheduling activities, or marking them as *done*, anytime a
```

     :align: center
     :alt: Custom filter pop-up with emphasis on the options for activities and due date.

Then, click into then next field and select :guilabel:`<=` from the drop-down menu. Selecting this
operator includes all activities with a due date up to, and including, the date selected in the next
field.

The third field can be left as today's date, or adjusted as needed.

Exclude unassigned leads
------------------------

Click :guilabel:`New Rule`. Click the first field for the new rule, and type `Salesperson` in the
:guilabel:`Search...` bar, or scroll to search through the list to locate it.

Click in the rule's second field, then select :guilabel:`is set` from the drop-down menu. Selecting
this operator excludes any leads not assigned to a specific salesperson.

Add a Sales team
----------------

Click :guilabel:`New Rule`. Click the first field for the new rule, and type `Sales Team` in the
:guilabel:`Search...` bar, or scroll to search through the list to locate it.

Click in the rule's second field, select :guilabel:`is in` from the drop-down menu. Selecting this
operator limits results to the sales teams selected in the next field.

> In the third field, select the desired sales team from the drop-down menu. Multiple teams can be
> selected in this field.

Reviewer 2: This is cool 👍 I didn't know you could add multiple teams here and it would count the parameters as part of an OR argument. I thought you had to create an **ANY** branch with 2+ choices in order to do this.

Suggestion to specify further what adding multiple teams in this field means, logically:
```suggestion
In the third field, select the desired sales team from the drop-down menu. Multiple teams can be
added in this field, where each parameter will be treated with an "or" (e.g. "any") operator in the search logic.
```

.. note::
   This filter is optional. To view results for the entire company, do **not** add this filter.

.. figure:: unattended_leads_report/configured-custom-rules.png
   :align: center
   :alt: An example of the Custom Filter pop-up window with all of the rules configured.

   An example of the **Add Custom Filter** pop-up window with all of the rules configured.

View results
------------

At the top of the :guilabel:`Add Custom Filter` form, there is an option to match :guilabel:`any` or
:guilabel:`all` of the rules. In order to properly run the report, only records that match **all**
of the following filters should be included. Before adding the filters, make sure :guilabel:`all` is
selected in this field.

.. image:: unattended_leads_report/all-custom-filter.png
   :align: center
   :alt: Example of overdue activities notification in the chatter of a lead.

After the filters are configured, click :guilabel:`Add`. The resulting report displays all leads
assigned to a salesperson where an activity is past due, or is due on the current date. The default
display is a bar graph, where the leads are grouped by *stage*.

> To group the results by salesperson, click the down arrow to the right of the search bar. Under the

Reviewer 2: missing guis here I think?

:guilabel:`Group By` heading, select :guilabel:`Salesperson`.

.. note::
   The option to group by :guilabel:`Sales Team` is also available under the :guilabel:`Group By`
   heading.

> To change to a *list* view, click the :guilabel:`≣ (bars)` icon in the top-right corner of the

Reviewer 2: Is **(bars)** what that icon is actually called? 
Hint 👇 
![Screenshot 2024-03-11 at 3 08 31 PM](https://github.com/odoo/documentation/assets/5496809/590ec04c-c142-417a-8f2d-fc8f6487b818)

screen.

.. tip::

>    Clicking the :guilabel:`(sliders) icon` opens a drop-down menu of additional columns that can be

Reviewer 2: Same here for **(sliders)**
![Screenshot 2024-03-11 at 3 11 09 PM](https://github.com/odoo/documentation/assets/5496809/b1a429c2-1909-4701-8697-df7cf3d83fd9)

Author: (FYI, it won't let me view these images, it says they are from a "private user", but I was able to figure out what you were trying to hint at :smiley: )

   added to the report.

   Some options that are beneficial for this report include:

   - :guilabel:`Activities`: the summary of the latest activity for this lead.
   - :guilabel:`Expected Closing`: the estimated date on which the lead will be won.
   - :guilabel:`Probability`: estimated success rate based on the stage.

  .. image:: unattended_leads_report/additional-options.png
     :align: center
     :alt: Custom filter pop-up with emphasis on the options for activities and due date.

.. _track_leads/unattended_leads_report/activity tracking:

> Activity tracking

Reviewer 2: Not sure this section is necessary since the [Activities](https://www.odoo.com/documentation/17.0/applications/productivity/discuss/overview/plan_activities.html) doc already exists and was pointed to much earlier in the article. 

Replacement ideas:
- a **seealso** admonition pointing to [Activities](https://www.odoo.com/documentation/17.0/applications/productivity/discuss/overview/plan_activities.html) doc (again, as a redundancy)
- best practices for sales team success if they want to implement activities tracking for their team.
- just leave blank and just remove what's there (can always add more later if we want)

Author: I think just the see also is best. At some point, it will probably be worth it to do one "Activities in the CRM" doc, which I know we discussed in the past, that can be linked here. Currently, the activities doc is about the function of the feature, but if we want to go into how to utilize them within the CRM best specifically, I think it would be best to keep that in one separate doc, instead of adding that on as a footnote. Thoughts?

Reviewer 2: > Thoughts?
Agree! I'll make a task.
